src/layers/src/Pane.js

```diff
@@ -2,12 +2,18 @@ import React, { memo, forwardRef } from 'react'
 import cx from 'classnames'
 import PropTypes from 'prop-types'
 import Box from 'ui-box'
-import usePaneAppearance from '../../theme/src/hooks/usePaneAppearance'
+import useStyleConfig from '../../hooks/use-style-config'
 
-const StringAndBoolPropType = PropTypes.oneOfType([
-  PropTypes.string,
-  PropTypes.bool
-])
+const pseudoSelectors = {
+  _hover: '&:hover',
+  _active: '&:active'
+}
+
+const internalStyles = {
+  transitionDuration: '150ms',
+  transitionProperty: 'box-shadow, transform',
+  transitionTimingFunction: 'cubic-bezier(0.0, 0.0, 0.2, 1)'
```

Comment:
Obvious non-blocker, but would be cool to push `transitionTimingFunciton` into the theme at some point. A lot more work ahead of us first :P